components/omega/src/infra/logging.cpp

```diff
@@ -0,0 +1,13 @@
+// Omega logging implementation file.
+
+#include "logging.h"
+
+void omega_log_init() {
```

Comment:
with the omega namespace, we don't need the omega_ prefix since we'll have the OMEGA:: delimiter. Also, we use CamelCase so this should be LogInit